### DISCUSSION TITLE
Empty drawings for some documents

### DISCUSSION THREAD

**Discussion by vkovenko** [pymupdf/PyMuPDF#2584] [Q&A]
Greetings. I found the following interesting issue while using pymupdf for svg extraction.
Method `get_drawings` returned empty list, while method `get_svg_image` worked out successfully. The question here is why get_drawings returned empty list if svg image is available (maybe just theoretical causes of such behavior)? 
Additional details:
PyMuPdf version: 1.21.1

**Comment by JorjMcKie** [comment]
The SVG image of a page primarily is an **_image_** trying to resemble a true appearance. It **_generates_** drawings for each text character if you don't prevent this with the option.
It is independent of the fact whether there also are original drawings on the page.

**Reply by JorjMcKie** [reply]
transformed this to a Discussions item. It is not an issue, but works as it should.
